### PR TITLE
Issue #69 - add fs2.Stream conversion

### PR DESCRIPTION
**Pull request by mindartur**
Modified version of proposed solution by @iravid in #69

### CODE REVIEW

**Comment by ghostdogpr** [comment]
Looks OK to me. @iravid you wanna double check?

**Comment by iravid** [comment]
This version looks good to merge to me. Great coverage @mindartur and love the chunk variant.

As a follow-up, I would suggest *always* using chunks, and maintaining the original chunking from the `fs2.Stream`. That is, use `evalTapChunk` (I think that exists - if not, then `evalMapChunk`), convert the `fs2.Chunk` to `zio.Chunk` and enqueue that.

File: interop-cats/shared/src/main/scala/zio/stream/interop/fsz2.scala [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package zio.stream.interop
+
+object fsz2 extends FS2Platform
```
Comment (ghostdogpr) [inline interop-cats/shared/src/main/scala/zio/stream/interop/fsz2.scala:3]: Btw shouldn't it be fs2z? 😄

Comment (mindartur) [reply interop-cats/shared/src/main/scala/zio/stream/interop/fsz2.scala:3]: yeah, right, I forgot it is scala**Z** :D

**Comment by mindartur** [comment]
@iravid @ghostdogpr Also, do you think that `toZStream(): ZStream[R, Throwable, A]` should always return `ZStream[Any, Throwable, A]`? 

I mean, `fs2.Stream` can't have any environment, right?

**Comment by ghostdogpr** [comment]
If you force the fs2 Stream to be on `Task` yeah the R can be just Any. But can’t you change it to `RIO[R, *]` so that it works for any R?

**Comment by mindartur** [comment]
> If you force the fs2 Stream to be on `Task` yeah the R can be just Any. But can’t you change it to `RIO[R, *]` so that it works for any R?

Changed `Task` to `RIO`

**Pull request merged**